react/week2/my-app/src/counter.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+// import ReactDOM from 'react-dom';
+import './index.css';
+//********************************************************************************************************************************************************* */
+
+class Counter extends React.Component {
+    state = { counter: this.props.initialCounter };
```

Comment:
For your use-case, I think it would be simpler to set the default value for state directly in the component: 
```suggestion
    state = { counter: 0 };
```